app/assets/javascripts/misc/pw-strength.js

```diff
@@ -35,7 +35,7 @@ function getFeedback(z) {
 
 function analyzePw() {
   const input = document.querySelector(
-    '#password_form_password, #update_user_password_form_password'
+    '#password_form_password, #reset_password_form_password, #update_user_password_form_password'
```

Comment:
Sure, @brendansudol has been meaning to clean this up. Can we do this in a separate PR?